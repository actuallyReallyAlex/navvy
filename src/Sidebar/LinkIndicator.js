import React from 'react'
import { Box } from 'grommet'

const LinkIndicator = () => {
  return (
    <Box
      background="blue"
      round="large"
      width="5px"
      style={{
        position: 'absolute',
        left: '0',
        right: '5px',
        top: '5px',
        bottom: '5px'
      }}
    />
  )
}

export default LinkIndicator
