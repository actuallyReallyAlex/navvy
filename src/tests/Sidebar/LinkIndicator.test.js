import React from 'react'
import LinkIndicator from '../../Sidebar/LinkIndicator'
import { shallow } from 'enzyme'

const defaultProps = {}

describe('<LinkIndicator />', () => {
  afterEach(() => {})

  test('Should render the <LinkIndicator /> component.', () => {
    const component = shallow(<LinkIndicator {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
