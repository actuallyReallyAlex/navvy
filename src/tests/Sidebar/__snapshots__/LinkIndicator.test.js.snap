// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkIndicator /> Should render the <LinkIndicator /> component. 1`] = `ShallowWrapper {}`;
